docs/design_proposals/event-api-v2.md

```diff
@@ -0,0 +1,119 @@
+# Title
+
+* Author: Nick Kubala
+* Design Shepherd: Any volunteers? :)
```

Comment:
I think we all should review this!

Reply:
+1 on review
It's an interesting question - what if a core member raises a design proposal? The role of a Design Sheperd is to provide communication with the core team. My opinion would be that we can remove this field.